tests/test_flobject.py

```diff
@@ -901,3 +901,19 @@ def get_child_nodes(node, nodes, type_list):
                 nodes[node_type] = node
                 if not type_list:
                     return
+
+
+@pytest.mark.fluent_241
```

Comment:
I don't think this needs to be marked as fluent_241 specific test. It should work for 23.2 (and may be 23.1) at least.

Reply:
@h-krishnan Thanks very much for resolving all issues.